src/malloc/mallocng/realloc.c

```diff
@@ -31,7 +40,7 @@ void *realloc(void *p, size_t n)
 		size_t base = (unsigned char *)p-start;
 		size_t needed = (n + base + UNIT + IB + 4095) & -4096;
 		new = g->maplen*4096UL == needed ? g->mem :
-			mremap(g->mem, g->maplen*4096UL, needed, MREMAP_MAYMOVE);
+			mremap_wrapper(g->mem, g->maplen*4096UL, needed, MREMAP_MAYMOVE);
```

Comment:
Why is this required?

Reply:
the function is variadic, and when compiling in genericjs the syscalls are genericjs, and it can't be used in malloc stuff because that is always asmjs. An alternative is to make it not variadic if the uses are all with the same arguments

Reply:
Let's go for not variadic.

Reply:
Pushed a new commit to make mremap not variadic.